index.js

```diff
@@ -15,8 +15,14 @@ module.exports = function browserifySize (name, options, callback) {
     if (err) return callback(err)
     bundle(dir, function (err, code) {
       if (err) return callback(err)
-      code = minify(code)
-      gzipSize(code, callback)
+      minify(code, function (error, minifiedCode) {
```

Comment:
* error -> err
* err === null -> !err
* return early instead of using an if/else